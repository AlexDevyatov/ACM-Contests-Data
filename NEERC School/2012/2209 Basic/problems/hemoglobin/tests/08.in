150
+2023
+1444
+2465
+2405
+1188
?3
-
-
+1136
-
+256
?1
?4
+1475
-
-
-
-
+1950
?1
?1
-
-
+124
-
+97
+2333
+2240
-
+1901
?1
?3
?3
+2568
?1
+2566
?2
-
?2
+1027
?1
+647
-
?3
+1700
?1
+985
?3
+2004
+789
-
+2861
+959
+838
+2606
?4
+924
?10
?13
?12
-
-
-
+2382
+1990
+2762
-
?1
-
+1813
+990
+1669
-
-
?10
+221
+2885
?4
?6
?12
-
+2889
+1313
?7
?8
+1584
-
-
?1
-
+2767
?11
+2574
+2283
+1844
?12
+2309
-
?7
-
+1163
-
?14
?15
-
?11
-
-
-
+563
-
+1956
?13
+572
?12
-
?9
-
-
-
+1883
+2218
?3
-
-
?4
?4
?10
?1
?9
+1627
-
+1131
-
-
?7
+1178
+1236
?8
-
?3
?3
?2
?1
-
?7
-
?5
-
+2749
