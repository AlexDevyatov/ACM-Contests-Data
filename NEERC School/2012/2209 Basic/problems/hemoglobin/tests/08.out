6058
1188
2405
1136
256
6188
1475
256
2465
1444
1950
1950
1950
2023
124
2240
1901
4331
4331
2568
5134
2566
4469
1027
647
5496
1700
3712
789
7264
16472
20803
20706
924
2606
838
2762
1990
1990
1669
990
18200
7301
11121
21306
2885
12438
14442
1584
1313
2889
2889
19287
22393
2309
13884
1844
1163
26045
28378
2283
19293
2574
2767
221
563
22586
20728
572
15687
1956
1813
2382
5060
2218
1883
6809
6809
16435
959
16338
1627
1131
959
13046
13559
1236
6043
6043
4039
1178
1178
13046
2861
8284
2004
